### PR TITLE
Adding some more logs

### PR DESCRIPTION
**Pull request by bkchr**
More logs are always better! Joke aside, these logs help to debug certain issues.

### CODE REVIEW

File: node/network/availability-distribution/src/requester/fetch_task/mod.rs [hunk 1]
```diff
@@ -326,6 +326,17 @@ impl RunningTask {
 		&mut self,
 		validator: &AuthorityDiscoveryId,
 	) -> std::result::Result<ChunkFetchingResponse, TaskError> {
+		gum::debug!(
```
Comment (eskimor) [inline node/network/availability-distribution/src/requester/fetch_task/mod.rs:329]: I think this is too noisy for debug. There are lots of chunks requested, trace would be better here.

File: node/network/bridge/src/network.rs [hunk 2]
```diff
@@ -161,6 +161,12 @@ impl Network for Arc<NetworkService<Block, Hash>> {
 		let peer_id = match peer {
 			Recipient::Peer(peer_id) => Some(peer_id),
 			Recipient::Authority(authority) => {
+				gum::debug!(
```
Comment (eskimor) [inline node/network/bridge/src/network.rs:164]: Also very likely too noisy for debug.

File: node/network/bridge/src/network.rs [hunk 3]
```diff
@@ -196,6 +202,14 @@ impl Network for Arc<NetworkService<Block, Hash>> {
 			Some(peer_id) => peer_id,
 		};
 
+		gum::debug!(
```
Comment (eskimor) [inline node/network/bridge/src/network.rs:205]: Same.

**Review by eskimor — COMMENTED** [review]
What is the issue that is being debugged?

**Comment by bkchr** [comment]
> What is the issue that is being debugged?

https://github.com/paritytech/cumulus/issues/1962

File: node/network/availability-distribution/src/requester/fetch_task/mod.rs (hunk 1 above)
Comment (bkchr) [inline node/network/availability-distribution/src/requester/fetch_task/mod.rs:329]:
```suggestion
		gum::trace!(
```

File: node/network/bridge/src/network.rs (hunk 3 above)
Comment (bkchr) [inline node/network/bridge/src/network.rs:205]:
```suggestion
		gum::trace!(
```

File: node/network/bridge/src/network.rs (hunk 2 above)
Comment (bkchr) [inline node/network/bridge/src/network.rs:164]:
```suggestion
				gum::trace!(
```

**Review by ordian — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

**Pull request merged**